### ISSUE TITLE
Tuple declaration not allowing pointer type for first element.

### ISSUE THREAD

**Issue by gopakumar-gg** [beefytech/Beef#1603]
```
			int x = 1, y = 2;
			(int *px, int *py) = (&x, &y); // Compilation ERROR.
			var p = (&x, &y); // Works fine.
			(int z, int *py) = (x, &y);  // Works fine.  First element is not a pointer.
```

**Comment by bfiete** [comment]
Just one note on `*` spacing- In C++, `int * a, b;`, `a` is `int*` and `b` is `int`. In Beef the `*` is actually part of the type itself so both are `int*`.

**Comment by bfiete** [comment]
Fixed at 48635c19394aa4e58b1b0b0dff687e220a52c414
